generator/src/main/kotlin/io/github/mscheong01/krotodc/util/StringExtensions.kt

```diff
@@ -0,0 +1,22 @@
+package io.github.mscheong01.krotodc.util
+
+fun String.snakeToCamelCaseNoRegex(): String {
```

Comment:
`fieldNameToJsonName` and `String.snakeToCamelCaseNoRegex` are both used to convert snake cased field names to json names (with the difference of handling characters after digits differently). Maybe we could commonize this into one method. If there's no reason to do the 'character after digit capitalizing' only for java field names, we could just use it for all camel case field name conversions. otherwise, the commonized method could have a parameter to enable that behavior explicitly.

Reply:
agree